### DISCUSSION TITLE
how to call a function when specific tab pane is clicked in antd?

### DISCUSSION THREAD

**Discussion by sai010101** [ant-design/ant-design#31526] [Q&A]
My Antd has a Tab in which there are 3 TabPane. I need to call a function if a specific TabPane is clicked.

A Parameter called "OnTabClick" exist only for the whole Tab (that contains tabpane) but not for the TabPane

**Comment by iseekTo** [comment]
Do you want to call a function after clicking on a specific tabPane?

**Reply by sai010101** [reply]
@iseekTo Yes
